apps/examples/testcase/le_tc/kernel/itc_environ.c

```diff
@@ -45,7 +45,7 @@
 * Preconditions         :none
 * Postconditions        :none
 */
-static void itc_environ_setenv_p(void)
```

Comment:
We check `_p` and `_n` internally in the automation tool. Do you have any reason to change?

Reply:
> We check `_p` and `_n` internally in the automation tool. Do you have any reason to change?

In some functions the _p and _n are at the middle and others are at the end of the function name, this is not identified properly in the TCM tool. So, I have changed the names as _pos and _neg at the end of the function for better identification and uniformity.